pygmt/src/select.py

```diff
@@ -128,6 +131,28 @@ def select(data=None, outfile=None, **kwargs):
         and **+i** reverses the tests to pass record with *z* value NOT in the
         given range. Finally, if **+c** is not used then it is automatically
         incremented for each new ``z_subregion`` argument, starting with 2.
+    dist2pt : str
+        *pointfile*\|\ *lon*/*lat*\ **+d**\ *dist*.
+        Pass all records whose locations are within *dist* of any of the
+        points in the ASCII file *pointfile*. If *dist* is zero, the 3rd
+        column of *pointfile* must have each point’s individual radius of
+        influence. If you only have a single point you can specify
+        *lon*/*lat* instead of *pointfile*. Distances are Cartesian and in
+        user units.
+    dist2line : str
+        *linefile*\ **+d**\ *dist*\ [**+p**].
+        Pass all records whose locations are within *dist* of any of the
+        line segments in the ASCII multiple-segment file *linefile*.
```

Comment:
I am wondering, whether all users (especially non GMT users) are familiar with the term "multiple-segment file"?

Reply:
Could link to https://docs.generic-mapping-tools.org/6.4/cookbook/file-formats.html#optional-segment-header-records perhaps?

Reply:
I added this link in commit https://github.com/GenericMappingTools/pygmt/pull/2466/commits/87138eab42f51b02126b52e7731d7774db841128.